pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/AddSchemaCommand.java

```diff
@@ -97,8 +70,8 @@ public String getName() {
   @Override
   public String toString() {
     String retString = ("AddSchema -controllerProtocol " + _controllerProtocol + " -controllerHost " + _controllerHost
-        + " -controllerPort " + _controllerPort + " -schemaFile " + _schemaFile + " -override " + _override + " _force "
-        + _force + " -user " + _user + " -password " + "[hidden]");
+        + " -controllerPort " + _controllerPort + " -database " + _database + " -schemaFile " + _schemaFile
+        + " -override " + _override + " _force " + _force + " -user " + _user + " -password " + "[hidden]");
```

Comment:
Consider adding a method in the base class to print the info managed in the base class, then only append extra fields here. Currently it doesn't cover all the supported fields